packages/frontend/src/components/SearchInput/index.tsx

```diff
@@ -17,7 +19,9 @@ export default function SearchInput({
         size="lg"
         fontSize="md"
         onChange={onChange}
+        defaultValue={searchValue}
         placeholder="Search"
+        autoFocus
```

Comment:
Why do we want to auto focus on the search bar ah? Is it the intended UX?

Reply:
cus when the user types an input in the flow, it loses focus after which was something that resulted because of the `SearchInput` being a custom component